OpenRA.Mods.RA/Activities/Infiltrate.cs

```diff
@@ -9,27 +9,25 @@
  */
 #endregion
 
+using OpenRA.Activities;
 using OpenRA.Mods.Common.Activities;
 using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.RA.Traits;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.RA.Activities
 {
 	class Infiltrate : Enter
 	{
 		readonly Actor target;
-		readonly Stance validStances;
+		readonly Infiltrates trait;
```

Comment:
Please rename this from `trait` to `infiltrates` to match our naming conventions.